### DISCUSSION TITLE
How to find vertex of LineString nearest to a coordinate?

### DISCUSSION THREAD

**Discussion by marcellangmaier** [NetTopologySuite/NetTopologySuite#711] [Q&A]
### Description
If I have a LineString with several vertices, how can I find the vertex nearest to a given coordinate?
The DistanceOp NearestPoints method returns calculated coordinates of points on the line but not only vertices.

**Comment by szlq** [comment]
Perhaps you can calculate the distance between a given point and each vertex on the LineString, and then find the smallest one
